utils/provided-goi-analysis-functions.R

```diff
@@ -147,53 +148,61 @@ plot_goi_expression_sina <- function(normalized_sce,
   #   normalized_sce: normalized SingleCellExperiment object
   #   goi_list: data frame with genes of interest relevant to the data in the
   #                 SingleCellExperiment object
-  #   ensembl_id_column: name of the column with the ensembl gene identifiers
+  #   sce_rownames_column: name of the column with the SCE rowname identifiers
   #                              used for mapping
-  #   gene_symbol_column: name of the column with the gene symbols that would be
-  #                              used for plotting if provided
+  #   optional_plotting_column: name of the column with the identifiers that
+  #                             would be used for plotting if provided
+  #   use_rownames: indicates whether or not the SCE rowname identifiers should
+  #                 be used when plotting; default is TRUE
   #
   # Returns:
   #   sina_expression_plot: sina plots displaying the logcounts expression
   #                         values for each gene of interest symbol relative to
   #                         the mean logcounts expression of all the genes of
   #                         interest
   
-  # Turn the ensembl and gene symbol column names into symbols for use when
-  # subsetting
-  ensembl_id_column_sym <- rlang::sym(ensembl_id_column)
+  # Turn the SCE rownames into symbols for use when subsetting
+  sce_rownames_column_sym <- rlang::sym(sce_rownames_column)
   
   # Run the `prepare_expression_df` function
   expression_means_df <- prepare_expression_df(normalized_sce,
                                                goi_list,
-                                               ensembl_id_column)
+                                               sce_rownames_column)
+  
+  # Check for optional plotting identifiers if `use_rownames` is FALSE
+  if(use_rownames == FALSE && is.null(optional_plotting_column)){
+    stop("When `use_rownames` = FALSE, a column name that contains optional 
+    plotting identifiers should be provided. Please provide this column name or
+    implement `use_rownames` = TRUE.")
+  }
```

Comment:
```suggestion
  if(!use_rownames && is.null(optional_plotting_column)){
    stop("When `use_rownames = FALSE`, a column name that contains optional 
    plotting identifiers should be provided. Please provide this column name or
    implement `use_rownames = TRUE`.")
  }
```
Just cleaning up and changing some formatting.